engines/gob/detection/tables_playtoons.h

```diff
@@ -337,7 +337,7 @@
 	kFeatures640x480,
 	"intro2.stk", 0, 0
 },
-{ // Supplied by indy4fan in bug report #13100. Orig title: "Der Mandarin-Prinz"
```

Comment:
That's the nickname used in the bugreport

Reply:
I know that the username on Discord is Indy4-Fan. And that is mostly used by the user.

Reply:
Besides the fact that the usefulness of these changes is debatable, usernames can be changed. If in the future you decide to use a different username, are you going to change all the occurrences in the detection tables?